tests/sentry/snuba/metrics/test_metrics_layer/test_metrics_enhanced_performance.py

```diff
@@ -1338,6 +1339,118 @@ def test_team_key_transactions_my_teams(self):
+                name=TransactionMRI.DURATION.value,
+                tags={"transaction": transaction},
+                timestamp=(now - timedelta(minutes=idx)).timestamp(),
+                value=value,
+                use_case_id=UseCaseKey.PERFORMANCE,
+            )
+
+        metrics_query = MetricsQuery(
+            org_id=self.organization.id,
+            project_ids=[self.project.id],
+            select=[
+                MetricField(
+                    op="team_key_transaction",
+                    metric_mri=str(TransactionMRI.DURATION.value),
+                    params={
+                        "team_key_condition_rhs": [
+                            (self.project.id, "foo_transaction"),
+                        ]
+                    },
+                    alias="team_key_transactions",
+                ),
+                MetricField(
+                    op="p95",
+                    metric_mri=str(TransactionMRI.DURATION.value),
+                    alias="p95",
+                ),
+            ],
+            start=now - timedelta(hours=1),
+            end=now,
+            granularity=Granularity(granularity=3600),
+            limit=Limit(limit=50),
+            offset=Offset(offset=0),
+            groupby=[
+                MetricGroupByField(
+                    field=MetricField(
+                        op="team_key_transaction",
+                        metric_mri=str(TransactionMRI.DURATION.value),
+                        params={
+                            "team_key_condition_rhs": [
+                                (self.project.id, "foo_transaction"),
+                            ]
+                        },
+                        alias="team_key_transactions",
+                    )
+                ),
+                MetricGroupByField("transaction"),
+            ],
+            orderby=[
+                OrderBy(
+                    field=MetricField(
+                        op="team_key_transaction",
+                        metric_mri=str(TransactionMRI.DURATION.value),
+                        params={
+                            "team_key_condition_rhs": [
+                                (self.project.id, "foo_transaction"),
+                            ]
+                        },
+                        alias="team_key_transactions",
+                    ),
+                    direction=Direction.DESC,
+                ),
+                OrderBy(
+                    field=MetricField(
+                        op="p95",
+                        metric_mri=str(TransactionMRI.DURATION.value),
+                        alias="p95",
+                    ),
+                    direction=Direction.DESC,
+                ),
+            ],
+            where=[
+                MetricConditionField(
+                    lhs=MetricField(
+                        op="team_key_transaction",
+                        metric_mri=str(TransactionMRI.DURATION.value),
```

Comment:
just replace `s/str(TransactionMRI.DURATION.value),/TransactionMRI.DURATION.value,`

Reply:
what do you mean?

Reply:
do you mean mypy right?

Reply:
ignore it. It's sed like style of replace. But we can skip it. Cause i feel mypy is causing this error.